### DISCUSSION TITLE
Storage account access

### DISCUSSION THREAD

**Discussion by hofmannma** [polymind-inc/acmebot#374] [Ideas]
Access to the storage account is via storage account access key which need to be configured as an application setting in the Azure function. The Azure function uses already a managed identity. How about using a managed identity to access the storage account vs. access key?

The Azure function is deployed with dependency for WEBSITE_RUN_FROM_PACKAGE pointing to https://shibayan.blob.core.windows.net/azure-keyvault-letsencrypt/v3/latest.zip. This creates an external dependency. Is it possible to setup the function without external dependency?

**Comment by shibayan** [comment]
Since the Storage Account used by Acmebot only contains the information necessary to execute the function, I don't think there is any motivation to actively use Managed Identity.

It is also possible to manually deploy the zip that is published in the URL. To ensure that Acmebot always delivers the latest features and bug fixes, it is strongly recommended that you deploy from a URL.
